sphinx_multiversion/main.py

```diff
@@ -173,6 +174,28 @@ def main(argv=None):
         action="store_true",
         help="dump generated metadata and exit",
     )
+    parser.add_argument(
+        "--add-dev",
+        action="store_true",
+        help="build development version from current directory",
```

Comment:
This is not used anywhere or am I missing something?

Reply:
Yes, my bad, it is redundant at this point. Removed.